pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -347,21 +347,23 @@ For more information about DataWeave support for Flex Gateway, see xref:gateway:
 [[use-custom-parameters-in-rust-source-code]]
 == Use Custom Parameters in Rust Source Code
 
-After you modify the `gcl.yaml` file, run the `build-asset-files` command to propagate the changes to the `src/generated/config.rs` rust file and the JSON schema for the UI:
+After you modify the `gcl.yaml` file, run one of the following commands to propagate the changes to the `src/generated/config.rs` rust file and the JSON schema for the UI:
```

Comment:
I'm not seeing the preceding text, so maybe it's explained. But I'm confused about "JSON schema for the UI". Which UI? It it referring to API Manager UI? Or VS Code UI? Or something else?

Reply:
API Manager UI added